identity-issuance/src/main/java/com/android/identity/issuance/funke/FunkeUtil.kt

```diff
@@ -3,32 +3,38 @@ package com.android.identity.issuance.funke
 import com.android.identity.crypto.Algorithm
 import com.android.identity.crypto.Crypto
 import com.android.identity.flow.server.FlowEnvironment
+import com.android.identity.issuance.IssuingAuthorityException
 import com.android.identity.securearea.CreateKeySettings
 import com.android.identity.securearea.KeyInfo
 import com.android.identity.securearea.SecureArea
+import com.android.identity.util.Logger
 import com.android.identity.util.toBase64Url
+import io.ktor.client.HttpClient
+import io.ktor.client.request.headers
+import io.ktor.client.request.post
+import io.ktor.client.request.setBody
+import io.ktor.client.statement.readBytes
+import io.ktor.http.HttpStatusCode
 import kotlinx.coroutines.sync.Mutex
 import kotlinx.coroutines.sync.withLock
 import kotlinx.datetime.Clock
-import kotlinx.serialization.json.JsonObject
 import kotlinx.serialization.json.JsonPrimitive
 import kotlinx.serialization.json.buildJsonObject
 import kotlin.random.Random
-import kotlin.time.Duration.Companion.days
-import kotlin.time.Duration.Companion.seconds
 
 internal object FunkeUtil {
-    // Endpoint for PID issuer, could be /c or /c1
+    // To use generic OpenID4VCI issuer, switch USE_AUSWEIS_SDK to false
+    //const val BASE_URL = "http://localhost:8080/server/openid4vci"
```

Comment:
hardcoded URL here is on the way out, it will be a part of the config, I just don't want to step on Dritan's work.

One URL is our own OpenID4VCI, the other is Funke